Entrega/Extra02.js

```diff
@@ -0,0 +1,8 @@
+var ano = Number(prompt("Digite um ano e te diremos se ele é ou não bissexto:"))
+if (ano % 100 == 0 && ano % 400 == 0){
```

Comment:
aqui é o contrario, amiga, se o resto da divisão por 100 for 0 e for multiplo de 4, ele não poderá ser bissexto

no caso dele ser divisivel por 400, ja é bissexto direto.

Reply:
Então, prof. Eu comecei com o caso mais raro. O ano é bissexto, apesar de ser divisível por 100, desde que ele seja também divisível por 400. (ocultei o divisível por 4 da conta pq todo ano divisível por 400 é também por 4, mas daria pra ter deixado só o 400 tbm, já que todo número divisível por 400 tbm é divisível por 100 e por 4). 
Então o código primeiro considera bissextos os anos que são divisíveis por 400.
Depois considera não bissextos os anos divisíveis por 100 (consequentemente por 4) e que não são divisíveis por 400.
Por último, o caso mais simples, considera bissextos os anos que são divisíveis por 4 e não divisíveis por 100 nem por 400.